_docs_v7/Slope-Limiters-and-Shock-Resolution.md

```diff
@@ -0,0 +1,205 @@
+
+
+
+### Available Limiters
+
+| Type | Description | Notes |
+| --- | --- | --- |
+| `NONE`                  | No limiter                                      |  |
+| `BARTH_JESPERSEN`       | Barth-Jespersen                                 |  |
+| `VENKATAKRISHNAN`       | Venkatakrishnan                                 |  |
+| `VENKATAKRISHNAN_WANG`  | Venkatakrishnan-Wang                            |  |
+| `SHARP_EDGES`           | Venkatakrishnan with sharp-edge modification    | This limiter should not be used for flow solvers |
+| `WALL_DISTANCE`         | Venkatakrishnan with wall distance modification | This limiter should not be used for flow solvers |
+| `VAN_ALBADA_EDGE`       | Van Albada (edge formulation)                   | This limiter has been observed to not output limiter values when called using VOLUME_OUTPUT |
+
+<!-- ??? Van Albada. It's possible that we actually have this backward and Van Albada is the only limiter that works for other solvers??? -->
+<!-- We're currently investigating if Barth-Jespersen is Venkat with K=0 -->
+
+The default limiter is `VENKATAKRISHNAN`.
+
+### Limiter Parameters and Further Details
+
+The `VENKAT_LIMITER_COEFF` parameter is generally a small constant, defaulting to $$0.05$$, but its specific definition depends on the limiter being used.
+This is different than the small constant used to prevent division by zero, which is used in all these limiters.
+<!-- ^ I'm refering to: `static Type epsilon() {return std::numeric_limits<passivedouble>::epsilon();}` in CLimiterDetails.hpp -->
+
+For the `VENKATAKRISHNAN`, `SHARP_EDGES`, and `WALL_DISTANCE` limiters, the `VENKAT_LIMITER_COEFF` parameter refers to $$K$$ in $$\epsilon^2=\left(K\bar{\Delta} \right)^3$$, where $$\bar{\Delta}$$ is an average grid size.
+The $$K$$ parameter defines a threshold, below which oscillations are not damped by the limiter, as described by [Venkatakrishnan](https://doi.org/10.1006/jcph.1995.1084).
+Thus, a large value will approach the case of using no limiter with undamped oscillations, while too small of a value will slow the convergence and add extra diffusion.
+This value depends on both the mesh and the flow variable and thus should be reduced if the mesh is refined.
+<!-- maybe change wording from flow variable to "field variable being limited" -->
+
+<!-- ??? should we include the definition of the venkat limiter (Delta -, Delta +, eps function) from the paper ??? -->
+<!-- ^^^ then we would also need to define Delta - and Delta +, which gets tricky. -->
+
+<!-- ??? should this section on Ref_Elem_Length be included ??? -->
+<!-- so, \bar{\Delta} is actually config.GetRefElemLength(), which refers to RefElemLength -->
+Similarly, the parameter `REF_ELEM_LENGTH` controls $$\bar{\Delta}$$, but the behavior of the limiter should be controlled through `VENKAT_LIMITER_COEFF`.
+The `REF_ELEM_LENGTH` parameter is also used in the geometric factor of the `SHARP_EDGES` and `WALL_DISTANCE` limiters.
```

Comment:
This option no longer exists, we lump the reference length and limiter constant in the same setting.